README.md

```diff
@@ -5,46 +5,52 @@ awesome-mycroft-community
 
-# STT/TTS replacements and utilities
-## STT
-### Systems
-### Utilities
-## TTS
-### Systems
-### Utilities
+- **Open Voice OS**: Open Voice OS is a community powered Linux distribution purpose-built to showcase the power of Open Source Voice AI for a range of devices. [Project Link](https://www.openvoiceos.com)
+- **Plasma Bigscreen**: An open UI stack for your own personal TV box using various open-source components like Plasma Bigscreen Launcher, Mycroft AI and Libcec based on KDE Neon Image. [Project Link](https://plasma-bigscreen.org/)
+- **Chatterbox**: The smart speaker anyone can build and teach, It's a smart speaker, like an Amazon Echo, but focused on education and privacy. The educational smart speaker to teach kids about coding, artificial intelligence and privacy. [Project Link](https://hellochatterbox.com/)
+- **Mycroft ROS**: This package provides capability to run the Text-To-Speech, Speech-To-Text and Skill functionality of Mycroft AI as ROS nodes providing topics, services and actions for communication. [Project Link](http://wiki.ros.org/mycroft_ros)
 
-# Wake word listeners and utilities
+### Mycroft Frameworks
+**_A list of frameworks built to extend Mycroft's capabilities beyond the default behavior which can be used standalone or in conjuction with each other_**
 
-# Skill management
+- **Lingua Franca**: Mycroft's multilingual text parsing and formatting library. A framework that is adopted as the common language between speakers with different native tongues. [Project Link](https://github.com/MycroftAI/lingua-franca)
+- **Hivemind**: HiveMind is a community-developed superset or extension of Mycroft, the open-source voice assistant. With HiveMind, you can extend one (or more, but usually just one!) instance of Mycroft to as many devices as you want, including devices that can't ordinarily run Mycroft! [Project Link](https://github.com/JarbasHiveMind/HiveMind-core)
+- **Mycroft GUI Framework**: Mycroft-GUI is an open source visual and display framework for Mycroft running on top of KDE Plasma Technology and built using Kirigami a lightweight user interface framework for convergent applications which are empowered by Qt. [Project Link](https://github.com/MycroftAI/mycroft-gui)
 
-# GUI
+### Plugins, STT / TTS Alternatives, Containers & Packages
+**_A list of Plugins, STT, TTS and alternative services that allow you to change the default behaviour of mycroft in terms of supported voice engines, supported backends, etc_**
 
-# HiveMind
+- **DeepSpeech**: DeepSpeech is an open-source Speech-To-Text engine, using a model trained by machine learning techniques based on Baidu's Deep Speech research paper. Project DeepSpeech uses Google's TensorFlow to make the implementation easier. [Project Link](https://github.com/mozilla/DeepSpeech)
```

Comment:
this is also not mycroft specific, would it make more sense to link a mycroft plugin for deep speech instead? or even better, link both